lib/src/main/kotlin/com/outdatedversion/survival/command/TimeStampCommand.kt

```diff
@@ -0,0 +1,55 @@
+package com.outdatedversion.survival.command
+
+import co.aikar.commands.BaseCommand
+import co.aikar.commands.annotation.CommandAlias
+import co.aikar.commands.annotation.CommandCompletion
+import co.aikar.commands.annotation.Default
+import com.outdatedversion.survival.Plugin
+import net.md_5.bungee.api.ChatColor
+import org.bukkit.Bukkit
+import org.bukkit.NamespacedKey
+import org.bukkit.entity.Player
+import org.bukkit.persistence.PersistentDataType
+import java.time.ZoneId
+import java.util.*
+// Now using Annotation Command Framework (ACF)
+@CommandAlias("timezone|tz")
+class TimeStampCommand(main: Plugin): BaseCommand() {
+  private val main: Plugin
+
+    @Default
+    @CommandCompletion("@timeZoneIDS") // Gets list from Plugin.kt (Main)
+  fun handleCommand(player: Player, vararg message: String) {
```

Comment:
I think this would be a cleaner way with ACF:
```suggestion
  fun handleCommand(player: Player, @Single @Optional zone: String?) {
```
With this, you'd convert the conditions below to check for `null` instead of the array's `size`

Reply:
Done! Will take effect next commit.